src/Frontend.hs

```diff
@@ -204,7 +204,7 @@ aulaSpace space =
        Page.viewIdeas  space
   :<|> Page.viewIdea
   :<|> Page.editIdea
-  :<|> Page.commentIdea locSpace
+  :<|> Page.commentIdea
```

Comment:
good point.  i think though that the way to resolve this is not to artificially use every part of the uri path, but to prune the path until it only contains useful parts.  see also: #302.